### ISSUE TITLE
Need to press TAB twice to open dropdown component

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#3823]
## Environment

> Operating system

Mac OS 10.14.6 (18G84)

> Browser
Chrome
> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

"carbon-components": "9.57.0",
"carbon-components-react": "6.81.4",
"carbon-icons": "7.0.7",

> What did you expect to happen?
Drop down get open at once when we are tabbing through it. As drop down in IBM cloud common filter
> What happened instead?

Need to press tab twice to open the dropdown
![image](https://user-images.githubusercontent.com/46310374/63683489-1c3c6080-c818-11e9-9f1e-ba59536e10eb.png)

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one

Login to the following url
https://cloud.ibm.com/security-advisor#/notifications
2. Step two
Click add notification button
3. Step three

Use keyboard tab to navigate method dropdown.
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by reeteshsingh93** [comment]
Any update on this?

**Comment by emyarod** [comment]
I believe this was resolved by https://github.com/carbon-design-system/carbon-components-react/pull/2376 in v6.115.1

Can you see if upgrading will resolve this issue for you? if you could reproduce the issue in a code sandbox for easier testing that would be great as well

**Comment by reeteshsingh93** [comment]
This worked for us. thanks for your support...
